src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,46 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //test ManufacturerService
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        // Manufacturer create test
+        Manufacturer testManufacturer =
+                manufacturerService.create(new Manufacturer("Kia", "South Korea"));
+        System.out.println(testManufacturer);
+        // Manufacturer get test
+        System.out.println(manufacturerService.get(2L));
+        // Manufacturer delete test
+        manufacturerService.delete(3L);
+        // Manufacturer update test
+        manufacturerService.update(new Manufacturer(2L,"Tesla", "USA"));
+        // Manufacturer getAll test
+        List<Manufacturer> allManufacturers = manufacturerService.getAll();
+        allManufacturers.stream().forEach(System.out::println);
```

Comment:
IDE should have highlighted it too
```suggestion
        allManufacturers.forEach(System.out::println);
```